google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -229,6 +235,45 @@ public EnhancedBigtableStub(EnhancedBigtableStubSettings settings, ClientContext
     readModifyWriteRowCallable = createReadModifyWriteRowCallable();
   }
 
+  private FlowController setupBulkMutationFlowController(BigtableBatchingCallSettings settings) {
```

Comment:
I wonder if we can split this up? I would like to keep the flow controller instantiation here, but would like to keep the default setting computation in a *Settings class. Also if are going to provide default thresholds, then we should set them during instantiation of stub settings so that users can print the defaults and see what they are. Finally the defaults should be documented